lib/utils.pm

```diff
@@ -163,19 +164,23 @@ sub integration_services_check_ip {
     my $vmname = console('svirt')->name;
     my $ips_host_pov;
     if (check_var('VIRSH_VMM_FAMILY', 'hyperv')) {
-        $ips_host_pov = console('svirt')->get_cmd_output(
-            'powershell -Command "Get-VM ' . $vmname . ' | Get-VMNetworkAdapter | Format-Table -HideTableHeaders IPAddresses"');
+        $ips_host_pov = console('svirt')
+          ->get_cmd_output('powershell -Command "Get-VM ' . $vmname . ' | Get-VMNetworkAdapter | Format-Table -HideTableHeaders IPAddresses"');
```

Comment:
these changes do not seem to relate to your commit. I suggest to revert them or keep them for another PR with a dedicated description.

Reply:
I did these changes to satisfy the travis tidy process. Do you have a better suggestion to make travis checking pass and do not need to touch these codes?

Reply:
**expect** is not used here, because we want more control with the procedure during the interactive. With expect command, all the interactive will be handled by the command itself, there's no way to control it from test script level. 

It is not a big problem currently, but in future if we want to extend the function, eg. make additional action according to some certain prompt message, it will be difficult. And as an addition, we also want to use _type_string_ and _send_key_ explicitly according to test case develper's choice.